server/validation/data-paths.ts

```diff
@@ -17,6 +18,8 @@ export enum DataPaths {
   RESOURCES_DISCORD_ACCESS = "resources/discord-access",
   RESOURCES_FORMS = "resources/forms",
   RESOURCES_IMPORTANT_LINKS = "resources/important-links",
+  RESOURCES_BUDGETS = "resources/budgets",
+  RESOURCES_MEETINGS = "resources/meetings",
```

Comment:
if we're doing this as part of this PR, can we also delete the `DOCUMENTS_BUDGETS` and `DOCUMENTS_MEETINGS` links these replace?